CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
-cmake_minimum_required(VERSION 3.24)
+cmake_minimum_required(VERSION 3.10)
 project(gravity_defied_cpp C CXX)
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
There is no need to force this option. You can pass it via` -D` on the command line.